### PR TITLE
[openebs] update charts to latest 0.8.1 release (#11650)

### PR DESCRIPTION
**Pull request by prateekpandey14**
- Updated README
  - Updated chart version
  - Updated values.yaml
  - Updated questions.yaml
  - Added liveness probe in all control-plane component templates
  - Change NDM update strategy to 'RollingUpdate'

Signed-off-by: prateekpandey14 <prateekpandey14@gmail.com>

### CODE REVIEW

**Comment by prateekpandey14** [comment]
@guangbochen Can you please review this ...thanks

File: proposed/openebs/latest/questions.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 categories:
 - storage
+labels:
+  io.rancher.certified: partner
```
Comment (guangbochen) [inline proposed/openebs/latest/questions.yml]: Hi @prateekpandey14 can u please share who is your contact person in rancher of the partner network, if not you may also contact us via https://rancher.com/partners/, thanks.

File: proposed/openebs/latest/questions.yml [hunk 2]
```diff
@@ -56,7 +58,7 @@ questions:
     type: string
     label: Node Disk Manager Image
   - variable: ndm.imageTag
-    default: "v0.2.0"
+    default: "v0.3.0"
```
Comment (kmova) [inline proposed/openebs/latest/questions.yml]: @prateekpandey14 Can you update this one to 0.3.1 please?

Comment (prateekpandey14) [reply proposed/openebs/latest/questions.yml]: Done !

File: proposed/openebs/latest/questions.yml (hunk 1 above)
Comment (prateekpandey14) [reply proposed/openebs/latest/questions.yml]: /cc @muratkars

Comment (muratkars) [reply proposed/openebs/latest/questions.yml]: Hi @guangbochen - our contact is Alex Belak (Director Strategic Alliances).

Comment (prateekpandey14) [reply proposed/openebs/latest/questions.yml]: @guangbochen PTAL thanks.

**Comment by kmova** [comment]
@guangbochen -- do you recommend any further changes on this PR? Should we remove the `partner` section for now - if that is blocking the merge?

**Comment by guangbochen** [comment]
I will review this PR before merging it, thanks for letting me know @kmova .
And for the `partner` label maybe we can add it in a separate PR if this is still in progress, thanks.

**Comment by kmova** [comment]
Looks like there is some epaper work to be completed to get the partner done. (cc: @muratkars). I like the idea of sending those partner status change in a separate PR. Thanks again as always for your help @guangbochen

**Comment by prateekpandey14** [comment]
@guangbochen PTAL , have remove the partner label for time being and will add in separate PR

**Comment by muratkars** [comment]
Hi @guangbochen could you please check with Alex and help to get the PR merged? Thanks!

**Comment by guangbochen** [comment]
@muratkars thanks for bear with me, because we have PR frozen due to rancher 2.2 release but I will push this to be merged AEAP, thanks.

**Comment by guangbochen** [comment]
Hi @prateekpandey14 can u please submit a new PR to the `dev` branch and close this one, we will push the new updates to the master release once it has been tested in the `dev` branch by our QA, thanks for bear with us.

**Comment by prateekpandey14** [comment]
@guangbochen have changed the base branch to `dev` from `master` PTAL.

**Comment by guangbochen** [comment]
LGTM.

**Comment by muratkars** [comment]
Thanks @guangbochen !

**Comment by guangbochen** [comment]
Hi @prateekpandey14, @kmova you can now submit a new PR to add the `partner` label as Alex B confirms it, thank you.

**Pull request merged**